### ISSUE TITLE
Not Receiving Notifications for some Requests

### ISSUE THREAD

**Issue by speedyflipper** [lokenx/plexrequests-meteor#239]
Every now and then notifications aren't sent when requests are made. It's happening with both movies and tv shows.

If someone know where the logs are stored I can look and see if there any messages.

**Comment by lokenx** [comment]
Logging is sent to the console, as well as on the admin page, under the logs tab. 

If there's still an issue here let me know, and we'll re-open!

**Comment by speedyflipper** [comment]
I'm still having this issue. I've checked the logs and there is nothing. Previously I was using pushbullet, now I'm using IFTTT Maker. I requested two movies within one minute but only received one notification. The one notification was received at 4:39PM and there is nothing in the logs at that time. Below are all of the logs for today (6/30)

{"level":"warn","message":"[Not Found]: Couldn't update seasons for American Crime Story","timestamp":"2016-06-30T09:13:27.690Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for Nathan For You","timestamp":"2016-06-30T09:13:28.245Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for 10 Things You Don't Know About","timestamp":"2016-06-30T09:13:28.778Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for Angie Tribeca","timestamp":"2016-06-30T09:13:29.313Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for Harry's Law","timestamp":"2016-06-30T09:13:29.843Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for Mozart in the Jungle","timestamp":"2016-06-30T09:13:30.362Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for 11.22.63","timestamp":"2016-06-30T09:13:30.889Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for Seinfeld","timestamp":"2016-06-30T09:13:31.431Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for Impractical Jokers","timestamp":"2016-06-30T09:13:31.964Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for Conviction","timestamp":"2016-06-30T09:13:32.491Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for Race for the White House","timestamp":"2016-06-30T09:13:33.027Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for Younger","timestamp":"2016-06-30T09:13:33.556Z"}
{"level":"warn","message":"[Not Found]: Couldn't update seasons for Benders","timestamp":"2016-06-30T09:13:34.087Z"}
